modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/experience/components/ExperienceItem.js

```diff
@@ -95,70 +97,99 @@ const ExperienceItem = ({
 		>
 			<ClayList.ItemField expand>
 				<ClayButton displayType="unstyled" onClick={handleSelect}>
-					<strong className="text-truncate">
-						{experience.name}
-
-						{experience.hasLockedSegmentsExperiment && (
-							<>
-								<ClayIcon
-									className="ml-2 text-secondary"
-									onMouseEnter={() => setShowtoolTip(true)}
-									onMouseLeave={() => setShowtoolTip(false)}
-									ref={iconRef}
-									symbol="lock"
-								/>
-
-								{showtoolTip && (
-									<Popover
-										anchor={iconRef.current}
-										header={Liferay.Language.get(
-											'experience-locked'
-										)}
-									>
-										{Liferay.Language.get(
-											'edit-is-not-allowed-for-this-experience'
-										)}
-									</Popover>
+					<ClayLayout.ContentRow verticalAlign="center">
+						<ClayLayout.ContentCol
+							style={{flexShrink: 1, minWidth: 0}}
+						>
+							<ClayLayout.ContentSection>
+								<span className="text-truncate-inline">
+									<ClayTooltipProvider>
+										<span
+											className="font-weight-semi-bold text-truncate"
+											data-tooltip-align="top"
+											title={experience.name}
+										>
+											{experience.name}
+										</span>
+									</ClayTooltipProvider>
+
+									{experience.hasLockedSegmentsExperiment && (
+										<span>
+											<ClayIcon
+												className="text-secondary"
+												onMouseEnter={() =>
+													setShowtoolTip(true)
+												}
+												onMouseLeave={() =>
+													setShowtoolTip(false)
+												}
+												ref={iconRef}
+												symbol="lock"
+											/>
+
+											{showtoolTip && (
+												<Popover
+													anchor={iconRef.current}
+													header={Liferay.Language.get(
+														'experience-locked'
+													)}
+												>
+													{Liferay.Language.get(
+														'edit-is-not-allowed-for-this-experience'
+													)}
+												</Popover>
+											)}
+										</span>
```

Comment:
This is just a suggestion, but I think that extracting this to a component (on the same file), would made it more easy to read
```because
   of
      the
                 indentation
                                   hell
```

Reply:
Sure, you're right! I'll do it 👍